.github/workflows/e2e_tests.yml

```
@@ -4,139 +4,42 @@ on:
-                  {
-                    "title": "Result",
-                    "value": "failure"
-                  },
-                  {
-                    "title": "Job URL",
-                    "value": "${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}"
-                  }
-                ]
-              }
-            ]
+  runs-on: ubuntu-latest
```

actionlint (1.7.7)

"steps" section is missing in job "runs-on"

(syntax-check)